### PR TITLE
[new] lib redux-toastr

### PR DESCRIPTION
**Pull request by diegoddox**
The repo of this project is here https://github.com/diegoddox/redux-toastr

### CODE REVIEW

**Comment by diegoddox** [comment]
I cannot what I doing wrong! I have the right indent on my package.json but it fails on the test!

File: ajax/libs/redux-toastr/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      "fonts/*",
+      "css/*"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/diegoddox/redux-toastr.git"
+  },
+  "license": "MIT"
+}
```
Comment (PeterDaveHello) [inline ajax/libs/redux-toastr/package.json]: Missing newline char here.

**Comment by PeterDaveHello** [comment]
We need a higher popularity to host this lib.

File: ajax/libs/redux-toastr/package.json (hunk 1 above)
Comment (diegoddox) [inline ajax/libs/redux-toastr/package.json]: Thanks!

**Comment by diegoddox** [comment]
Ok thanks!

**Pull request closed without merging**